source/renderer/app/stores/WalletsStore.ts

```diff
@@ -1508,6 +1541,11 @@ export default class WalletsStore extends Store {
       this.actions.wallets.generateAddressPDFSuccess.trigger({
         walletAddress,
       });
+      this.stores.analytics.analyticsClient.sendEvent(
+        'Wallets',
```

Comment:
Yeah we'd benefit from const variables here. :)

Reply:
If they are unique, at least we can keep a reference of these actions somewhere.

But, the question is: is an action's name determined by the author and will it always be unique at all times? I'm guessing there's a reference on the other side that knows which actions to listen to, in which case it's better to keep a reference.

Reply:
Matomo doesn't work like PubSub, it doesn't need to know what are the correct action names. It accepts whatever values we push to its API. Here is an example dashboard it renders based on the data provided:

![image](https://user-images.githubusercontent.com/6684554/177303035-c8dc932f-c315-406a-ad88-cdfa0958134e.png)

In order to render such a dashboard, the only thing you need to configure in Matomo is some basic site config (title, URL, etc.).

It acts pretty much as logger - the main difference is that it is not used for troubleshooting but to understand the user behaviour and improve the UX. And I've never seen any codebase where log messages would be extracted to an enum or constants, normally they are just plain inline strings. 

I personally believe it makes sense to extract event categories to an enum to minimise a risk of a typo which would then result in two categories which in fact should be one, but for action names which are unique I would keep them as is.

Reply:
Then let's go with just categories. 👍

Reply:
Cool, I extracted categories to an enum called `EventCategories` earlier.